cms-common/src/main/java/io/commercetools/sunrise/cms/CmsService.java

```diff
@@ -15,15 +15,17 @@
      * Gets the content corresponding to the given CMS identifier for the first found given language.
      * @param locales the list of locales used to translate the message
      * @param cmsIdentifier identifier of the CMS entry field
-     * @return the {@code completionStage} of the content in the first found given language, or absent if it could not be found
+     * @return the {@code completionStage} of the content in the first found given language, or absent if it could not be found, or a
+     * {@link CmsServiceException} if there was problem during fetching response from CMS system.
```

Comment:
`if there was a problem when obtaining the content from the CMS` would be better IMO.
Some reasons:
- "fetch a response" is too concrete to a remote or external system, but actually a file-based system is also an acceptable CMS (even though really rudimentary).
- CMS contains "system" in the name, so system after it would be redundant ;)